### PR TITLE
fixed space issues

### PR DESCRIPTION
**Pull request by Tej-Singh-Rana**
regards,

### CODE REVIEW

File: content/en/docs/concepts/cluster-administration/flow-control.md [hunk 1]
```diff
@@ -414,7 +414,7 @@ When you enable the API Priority and Fairness feature, the kube-apiserver serves
   system,            system-nodes,   12,         0,                   system:node:127.0.0.1, 2020-07-23T15:26:57.179170694Z,
   ```
   
-  In addition to the queued requests, the output includeas one phantom line for each priority level that is exempt from limitation.
+  In addition to the queued requests, the output include as one phantom line for each priority level that is exempt from limitation.
```
Comment (sftim) [inline content/en/docs/concepts/cluster-administration/flow-control.md:417]: I believe the original author intended:
```suggestion
  In addition to the queued requests, the output includes one phantom line for each priority level that is exempt from limitation.
```

Comment (Tej-Singh-Rana) [reply content/en/docs/concepts/cluster-administration/flow-control.md:417]: should i change again or they will do?

File: content/en/docs/concepts/cluster-administration/addons.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ Add-ons in each section are sorted alphabetically - the ordering does not imply
...
 * [Nuage](https://github.com/nuagenetworks/nuage-kubernetes/blob/v5.1.1-1/docs/kubernetes-1-installation.rst) is an SDN platform that provides policy-based networking between Kubernetes Pods and non-Kubernetes environments with visibility and security monitoring.
 * [Romana](http://romana.io) is a Layer 3 networking solution for pod networks that also supports the [NetworkPolicy API](/docs/concepts/services-networking/network-policies/). Kubeadm add-on installation details available [here](https://github.com/romana/romana/tree/master/containerize).
-* [Weave Net](https://www.weave.works/docs/net/latest/kube-addon/) provides networking and network policy, will carry on working on both sides of a network partition, and does not require an external database.
+* [Weave Net](https://www.weave.works/docs/net/latest/kubernetes/kube-addon/) provides networking and network policy, will carry on working on both sides of a network partition, and does not require an external database.
```
Comment (Cweiping) [inline content/en/docs/concepts/cluster-administration/addons.md:37]: Different languages are updated together. Thx!

**Comment by jimangel** [comment]
@Tej-Singh-Rana the changes requested in reviews are normally completed by the PR author. Let me know if you have any questions! Thanks!

**Comment by Tej-Singh-Rana** [comment]
It's Okay, I don't have any question.

**Comment by zacharysarah** [comment]
@Tej-Singh-Rana 👋 Feel free to `/reopen` this PR when you're ready to incorporate reviewer feedback and rebase this PR.

/close

**Pull request closed without merging**